### PR TITLE
Updating statuses.py

### PR DESCRIPTION
**Pull request by toraigeki**
Added 3 quotes, some french ones 👀 (translated the best I could)

### CODE REVIEW

File: statuses.py [hunk 1]
```diff
@@ -81,4 +81,7 @@
    "Keep calm and give a hug.",
    "Start your day with a hug! 🤗",
    "Feel the presence of love wrapped up within a hug.",
+   "Better than esperanto, hugs are a universal language.",
+   "How can we renounce the cuddly habits, the comfort, the indolent well-being of life? - François René de Chateaubriand / Mémoires d’outre-tombe ",
```
Comment (achasveachas) [inline statuses.py:85]: In the quotes that are quoting someone, I like it better when the person being quoted is in their own line (see the quote on line 70 for example).

Also, there seems to have been something lost in translation? I'm not sure what the word "indolent" is meant to be here, are there maybe other suggested translations for the original French?

File: statuses.py [hunk 2]
```diff
@@ -81,4 +81,7 @@
...
+   "The hug has no timen it is salutary by day and by night. - From 'Le petit livre des gros câlins' (Little book of big hugs)",
```
Comment (achasveachas) [inline statuses.py:86]: Same comment as above regarding putting the source on a separate line.

Also, the word "salutary" seems a bit out of place here. Maybe "beneficial" or "helpful" would be better?

**Review by achasveachas — CHANGES_REQUESTED** [review]
Merci! These are great! 🤗 

A few minor changes and then I'll merge it!

File: statuses.py (hunk 1 above)
Comment (toraigeki) [reply statuses.py:85]: It's an old form of the same french word "**indolent**" (_plural here_), It's more of a reference to an easy going life, without trouble and pain, not "**lazy**" or something similar that can be of a negative connotation of a bad habit.
I could perhaps use the term "**indulging**" instead in the context, if it "indolent" doesn't seem fitting in English.

NOTE: The work is from 1850 so the use of such expression may have been different.

File: statuses.py (hunk 2 above)
Comment (toraigeki) [reply statuses.py:86]: I tried to keep it close to the original form of expression "**salutaire**" but, if a more common language is more befitting of the quote stacks the twitter bit already has then, sure, I could use "**soothing**" instead.

**Review by toraigeki — COMMENTED** [review]
Updating new changes based on previous review:
- Writing original author's names on different line from the quote.
- Editing translated words for alternative synonyms:
`Indolent` -> `indulging`.
`Salutary` -> `soothing`.

File: statuses.py (hunk 1 above)
Comment (achasveachas) [reply statuses.py]: I see. It still came across as clumsy to me, so I searched and found an English translation of the work ([here](https://www.poetryintranslation.com/PITBR/Chateaubriand/ChateaubriandMemoirsBookVIII.php#anchor_Toc117934906)) and it seems like the quote goes something like "How to forego life’s tender usages, comforts, and indolent well-being?"

This seems to be less about cuddles and more about comfortable habits, so maybe let's leave this one out?

File: statuses.py (hunk 2 above)
Comment (achasveachas) [reply statuses.py]: As someone who used to do a lot of Hebrew-English and Yiddish-English translation I relate to the struggle between trying to stay true to the original wording and trying to make the translation more approachable in the target language :)

"Soothing" is great! We'll go with that!

**Review by achasveachas — CHANGES_REQUESTED** [review]
As mentioned, let's remove the quote from Mémoires d’outre-tombe and leave the other two.

File: statuses.py (hunk 1 above)
Comment (toraigeki) [reply statuses.py]: Maybe it is, I focused on the "cuddly" habits after reading the response to comments in the original issue #12 regarding having the word "hug" in all quotes seeming like "compulsory". 
But sure, we can skip this one. I'll adjust the PR to it now.

**Comment by toraigeki** [comment]
Alright @achasveachas , as requested in the previous review, I dropped the François René's quote. I also sync-ed the repo, since it seems it had new updates so I added the quotes on new lines to avoid merge conflict.
Hopefully I've done things right.

**Review by achasveachas — APPROVED** [review]
Great, merci beaucoup! 🤗 

Let me know if you have a Twitter handle so I can give you a shout-out!

**Comment by toraigeki** [comment]
Thank you for your efforts and patience! I don't have twitter but it was nice contributing to the quote bot!

**Comment by achasveachas** [comment]
No, thank YOU! It's people like you who make it awesome 😊

**Pull request merged**